univdev/ww-timer
# Add Contribution Guidelines and Templates for Issues & PRs

## Overview

This Pull Request introduces the CONTRIBUTING.md file alongside templates for both Issues and Pull Requests to the project. These additions aim to streamline the contribution process, ensuring contributors can easily understand how to participate in the project and what information they need to provide when submitting issues or pull requests.

## Changes

- Added `CONTRIBUTING.md`: Contains guidelines for how to contribute to the project, including code of conduct, setting up the development environment, and the contribution process.
- Added `.github/ISSUE_TEMPLATE.md`: A template for submitting new issues, guiding contributors to provide necessary details such as a description of the issue, steps to reproduce, expected behavior, and screenshots if applicable.
- Added `.github/PULL_REQUEST_TEMPLATE.md`: A template for creating new pull requests, encouraging contributors to detail the changes made, the reason behind the changes, and any additional context that could help the review process.

## Rationale

The addition of these files is intended to make the contribution process more efficient and contributor-friendly. Clear guidelines and templates help maintain the quality of contributions and make it easier for both contributors and maintainers to communicate effectively.

## How to Test

- Please review the `CONTRIBUTING.md` file for clarity and completeness.
- Check the `ISSUE_TEMPLATE.md` and `PULL_REQUEST_TEMPLATE.md` files to ensure they cover all necessary information contributors should provide.
